courses/guidance/06.md

```diff
@@ -0,0 +1,26 @@
+---
+index: 6
+title: 6. Editing Componenets
+author: Sumaa Sayed
+headerDepth: 3
+---
+
+## 6. Tailoring Your Components
+
+### 6.1
+- For your application, you'll need to decide what controls will be appliable to your application, whihc ones need some tailoring, and which ones will not be applicable to your application.
+
+### 6.2 Editing Components
```

Comment:
might be good to state that for each status, there are some fields that must be populated before saving the control.